src/main/java/org/apache/commons/imaging/formats/jpeg/iptc/IptcParser.java

```diff
@@ -438,8 +477,8 @@ public int compare(final IptcRecord e1, final IptcRecord e2) {
                 }
                 bos.write(element.iptcType.getType());
 
-                final byte[] recordData = element.value.getBytes("ISO-8859-1");
-                if (!new String(recordData, "ISO-8859-1").equals(element.value)) {
+                final byte[] recordData = element.value.getBytes(DEFAULT_ENCODING);
+                if (!new String(recordData, DEFAULT_ENCODING).equals(element.value)) {
                     throw new ImageWriteException(
                             "Invalid record value, not ISO-8859-1");
```

Comment:
This should also use `DEFAULT_ENCODING

Reply:
Good catch, this way we won't forget changing the exception text :+1: